.github/workflows/run-magento-tests.yml

```diff
@@ -65,4 +68,5 @@ jobs:
     - name: Run Unit Tests
       run: |
         cd magento
+        sudo sed -i "/<string>allure.*/c\<directory>allure/allure.config.php</directory>" dev/tests/unit/phpunit.xml.dist
```

Comment:
What is this change for?

Reply:
In the latest magento 2.4.6, When we run the PHP unit test, it throws an error: `Error: Config file allure/allure.config.php doesn't exist`,
By adding this line, it fixes the error and run the PHPUnit test successfully.